src/modules/aws-ec2/shutdown-behavior/changeShutdownBehavior.ts

```diff
@@ -0,0 +1,20 @@
+import { EC2 } from "aws-sdk";
+import { logger } from "../../../logger";
+
+export const changeShutdownBehavior = async (
+  instanceId: string,
+  value: string
+) => {
+  const ec2 = new EC2();
+  const params = {
+    Attribute: "instanceInitiatedShutdownBehavior",
+    InstanceId: instanceId,
+    Value: value
+  };
+  try {
+    await ec2.modifyInstanceAttribute(params).promise();
+  } catch (error) {
+    logger.error(error.message);
+  }
```

Comment:
Another instance of `try/catch/log` without proper error handling. Probably safer to just let the error bubble up and be handled further up the callstack.